src/main/webapp/WEB-INF/views/UserPage.html

```diff
@@ -0,0 +1,388 @@
+                    <span class="pull-left" style="margin-right:8px; margin-top:-5px;">
+                        <img src="http://vev.ru/uploads/images/00/07/64/2011/10/12/c26b1b.jpg"
+                             class="img-responsive img-circle" title="UserName"
+                             alt="UserName" width="30px" height="30px"/>
+                    </span>
+                    <span>UserName</span>
+                    <b class="caret"></b></a>
+                    <ul class="dropdown-menu">
+                        <li>
+                            <div class="navbar-content">
+                                <div class="row">
+                                    <div class="col-md-5">
+                                        <img src="http://vev.ru/uploads/images/00/07/64/2011/10/12/c26b1b.jpg"
+                                             alt="AltText" class="img-responsive"
+                                             width="120px" height="120px"/>
+                                    </div>
+                                    <div class="col-md-7">
+                                        <span>Welcome back,</span>
+                                        <p class="text-muted">UserName!</p>
+                                        <div class="btn-group-vertical" style="margin-top: -30px">
+                                            <a href="#" class="btn btn-link btn-md"><i class="fa fa-user-o"
+                                                                                       aria-hidden="true"
+                                                                                       style="margin-right: 5px"></i>My
+                                                Account</a>
+                                            <a href="#" class="btn btn-link btn-md"><i class="fa fa-star"
+                                                                                       aria-hidden="true"
+                                                                                       style="margin-right: 5px"></i>Favourites</a>
+                                            <a href="#" class="btn-link btn btn-md"><i class="fa fa-question-circle-o"
+                                                                                       style="margin-right: 5px"></i>Help
+                                            </a>
+                                        </div>
+                                    </div>
+                                </div>
+                            </div>
+                            <div class="navbar-footer">
+                                <div class="navbar-footer-content">
+                                    <div class="row">
+                                        <div class="col-md-6 pull-right">
+                                            <a href="#" class="btn btn-default btn-block"><i class="fa fa-sign-out"
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') Replace this <i> tag by <em>. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=Web%3ABoldAndItalicTagsCheck)